db/migrate/20230329123641_change_permalink_from_products.rb

```diff
@@ -0,0 +1,5 @@
+class ChangePermalinkFromProducts < ActiveRecord::Migration[7.0]
+  def change
+    change_column :products, :permalink, :text
```

Comment:
Is this a deliberate example of irreversible migration??

Reply:
change_column is irreversible. To make it reversible we need to either define up and down method or define reversible inside change method.